aws-aurora/main.tf

```diff
@@ -21,7 +21,7 @@ resource "aws_security_group" "rds" {
     from_port   = "${var.port}"
     to_port     = "${var.port}"
     protocol    = "tcp"
-    cidr_blocks = ["${var.ingress_cidr_blocks}"]
```

Comment:
these do break my tf 0.11. just ran into a couple instances

Reply:
Moved the whole commit to the 0.12 breaking PR.